config/webpack/environment.js

```diff
@@ -62,4 +51,9 @@ Object.assign(fileLoader.use[0].options, {
   postTransformPublicPath: (p) => `window.rails_asset_host + ${p}`
 });
 
+// Whatever this is, it seems we don't need it.
+environment.loaders.delete('nodeModules')
+
+console.dir(environment.loaders, { depth: null })
```

Comment:
what is this doing?

Reply:
This is for debugging, I left it there by accident 😅